FEATURES.md

```diff
@@ -0,0 +1,35 @@
+# Feature Maturity
+
+Maturity of features in Istio indicate to users what they can [expect](https://istio.io/latest/about/feature-stages/) by using these
+features. In Istio, we track these requirements using a [template](features/feature_template.md). Once all
+requirements for promoting a feature to the next level are met, the [features](features.yaml)
+file may be updated to indicate this maturity level.
+
+## Features.yaml
+
+Features.yaml tracks the maturity of all Istio features. This is written
+in YAML so that it can be easily processed by any tooling that needs to
+track feature maturity. Each feature in Istio should have a features
+entry as described below.
+
+```
+features:
+  - name: "Protocols:HTTP1.1/HTTP2/gRPC/TCP"
+    level
+      - checkList: 
+        current:
+        promotionBefore:
+    state
+      - state:
+        begin:
+        details:
+```
+
+* **Name** indicates the name of the feature.
+* **level** indicates the maturity level of a feature.
+* **level.checklist** is the path to the checklist to track a feature's maturity.
+* **level.current** is the current maturity level of the feature. This can be `experimental`, `alpha`, `beta`, or `stable`. Experimental is an optional maturity level that can be skipped. For alpha, beta, and stable, all requirements for the previous levels must be met before a feature can meet the next level (i.e. a stable feature has met the requirements of both alpha and beta). There may be features for which a requirement is not applicable.
```

Comment:
Don't love level.current, maybe level.maturity or level.stage or something?

Reply:
level.maturity works for me.